inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/config.cpp

```diff
@@ -75,6 +86,11 @@ namespace {
             {{InferenceEngine::MultiDeviceConfigParams::KEY_MULTI_DEVICE_PRIORITIES , CommonTestUtils::DEVICE_CPU}}
     };
 
+    const std::vector<std::map<std::string, std::string>> autoinconfigs = {
```

Comment:
Thanks Maxim, but for this PR, we don't support such configuration like AUTO_*, we will create another PR to add config for AUTO. So you can see there is no config code for AUTO now, because I delete them all.